badge-maker/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "gh-badges",
-  "version": "2.2.1",
+  "name": "badge-maker",
+  "version": "3.0.0-rc1",
```

Comment:
* I've already published a package that does nothing at https://www.npmjs.com/package/badge-maker just to stake a claim on the name.
* I'm suggesting the first release is `-rc1` here, but I don't really have strong opinions on where it should be `-beta` or whatever..